src/go/k8s/cmd/main.go

```diff
@@ -334,13 +336,12 @@ func main() {
 
 		// Helm Release Controller
 		helmRelease := helmController.HelmReleaseReconcilerFactory{
-			Client:              mgr.GetClient(),
-			Config:              mgr.GetConfig(),
-			Scheme:              mgr.GetScheme(),
-			EventRecorder:       helmReleaseEventRecorder,
-			ClientOpts:          clientOptions,
-			KubeConfigOpts:      kubeConfigOpts,
-			NoCrossNamespaceRef: true,
+			Client:           mgr.GetClient(),
+			EventRecorder:    helmReleaseEventRecorder,
+			ClientOpts:       clientOptions,
+			KubeConfigOpts:   kubeConfigOpts,
+			FieldManager:     "redpanda-helmRelease-controller",
+			GetClusterConfig: ctrl.GetConfig,
```

Comment:
Metrics handler is missing.

https://github.com/redpanda-data/redpanda-operator/pull/43/files#diff-8659c00d96d07c10eb47505088baad8cb1d6563e41cfc08e532d214816a400efR347